node-mixin/alerts/alerts.libsonnet

```diff
@@ -41,9 +41,9 @@
           {
             alert: 'NodeFilesystemOutOfSpace',
             expr: |||
-                node_filesystem_avail{%(nodeExporterSelector)s,%(fsSelectors)s} / node_filesystem_size{%(nodeExporterSelector)s,%(fsSelectors)s} * 100 < 5
```

Comment:
It does not overlap. If a disk is 95% full without any changes, it will not trigger any of the predict_linear alerts.

This alert was created because filesystems are in general not happy if they are almost full. SSDs also like to have empty space for their housekeeping. (Strictly speaking, the latter is not per filesystem but per device. There are people who don't partition parts of their SSDs to guarantee that slack space. More commonly, people partition everything and expect users to usually not fill their filesystems to the brink.)

Thus, even if a filesystem usage is not growing, if there are still mutating operations happening, you probably want more than 5% free. Thus, this alert, and thus the naming with "out of space".

I'll change it anyway to "NodeFilesystemAlmostOutOfSpace" to be strictly true.